services/efs-guardian/Dockerfile

```diff
@@ -45,26 +45,41 @@ RUN --mount=type=cache,target=/var/cache/apt,mode=0755,sharing=private \
   && gosu nobody true
 
 # simcore-user uid=8004(scu) gid=8004(scu) groups=8004(scu)
-ENV SC_USER_ID=8004 \
-  SC_USER_NAME=scu \
+# simcore-efs-user uid=8006(efs) gid=8006(efs) groups=8006(efs)
+# The Guardian has access to the root directory of the EFS distributed file system
+# and can change the permissions if needed. User running services operates as a user with ID 8004.
+ENV SC_USER_ID=8006 \
+  SC_USER_NAME=efs \
```

Comment:
As I wrote in the other comment, I would suggest to use the ENV given here, that means they are defined in the container, and are accessible from the efs-guardian settings... and then also directly usable there. THis would improve the maintainability by a factor.

Reply:
I have added the variables to the Application settings. For now I will also leave them in the Dockerfile to be consistent - we can discuss in person.